src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -368,9 +368,9 @@ public PacketTransformer getRTCPTransformer()
      *
      * @return the current padding parameters list for {@link #dest}.
      */
-    List<SimulcastController> getSimulcastControllers()
+    List<AdaptiveTrackProjection> getAdaptiveTrackProjections()
     {
-        return simulcastControllers;
+        return adaptiveTrackProjections;
```

Comment:
we return the reference here but also modify it locally, so we could run into concurrent modification exception it seems?  maybe return a copy or an unmodifiable version so we can catch callers of this who try to change it?